content/en/platform/corda/5.1/reference/certificates.md

```diff
@@ -0,0 +1,84 @@
+| E2E Identity Trust Roots       | Trust root of certificates for session initiation keys.                                                                                                                                                | 256-bit PEM       |
+| Certificate for Corda releases | Trusted certificates for binary Corda release verification.                                                                                                                                            | X.509 Certificate |
+
+## Credentials
+
+Corda uses the following credentials:
+
+| Credential                                     | Description                                                                                                                                                                                                                                | Type                                  |
+| ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------- |
+| SSO Config credentials                         | SSO credentials used by the HTTP Gateway process. This is an API ID provided by Azure AD.                                                                                                                                                  | String                                |
+| Installation credentials creation              | Installation credentials with enough entropy created in Kubernetes etcd, Helm Chart, or another vault. These credentials are used for key derivation and cluster database secret encryption.                                                 | Passphrase and salt                   |
+| Client Kafka credentials for a specific worker | Kafka credentials (username and password) created at deployment time and stored by default in Kubernetes etcd as secret configuration. This value is passed to worker pods memory at runtime.                                              | High entropy unencrypted string       |
+| Kubernetes secrets creation via API            | The Cluster Adminstrator creates the secrets in Kubernetes etcd to be used by the Corda Helm charts. This is executed optionally when Helm Chart overrides are not used as an alternative.                                                |                                       |
+| Kubernetes Secrets configured in Helm charts   | The Cluster Administrator configures the Secrets as overrides in Helm charts. This is an alternative to creating Kubernetes Secrets via API.                                                                                                                     |                                       |
```

Comment:
```suggestion
| Kubernetes Secrets configured in Helm charts   | The Cluster Administrator configures the secrets as overrides in Helm charts. This is an alternative to creating Kubernetes secrets via API.                                                                                                                     |                                       |
```

Reply:
Capitalised in the Kubernetes docs so leaving as is.